### ISSUE TITLE
How to increase training rate.

### ISSUE THREAD

**Issue by SJHBXShub** [wushilian/CRNN_Attention_OCR#26]
Hi, wu
I found  changing this line(in model line 81) **loss = tf.reduce_mean(att_loss)**  to **loss = tf.reduce_sum(att_loss)** will greatly increase the training speed.  
The reason, I thought, is reduce_sum return a bigger loss, which will directly cause that every trainable parameter will be changed greatly. 

Brs
SJHB

**Comment by l00350898** [comment]
你好，你训练的时候超参分别是多少，最后收敛到多少呀？我训练的时候loss在0.14，精度到70%的时候loss会突然变成3.5左右，然后迭代也不收敛

**Comment by wushilian** [comment]
@SJHBXShub That is equivalent to increasing the learning rate

**Comment by wushilian** [comment]
@l00350898 turn the learning rate to 1e-5
